### DISCUSSION TITLE
Prowler dashboard remediation column

### DISCUSSION THREAD

**Discussion by kalbert718** [prowler-cloud/prowler#3854] [Q&A] [answered]
Does anyone know if it is possible to include the remediation column in the dashboard? I am not sure how to enable it on the latest version of Prowler, even though it was included in older versions.

**Comment by pedrooot** [comment, marked as answer]
Hey! @kalbert718 we are working on many new capabilities and features for the dashboard. On our to-do list we have this improvement noted. If you want to contribute and add this feature just tell me and I'll explain more to you about the field.
Thanks for using prowler! 🚀

**Reply by pedrooot** [reply]
btw you can check the new features added in Prowler 4.1 version! 
![Screenshot 2024-04-25 at 17 18 22](https://github.com/prowler-cloud/prowler/assets/56402503/dcf8cd31-fb88-4038-bde4-be6b8097af7f)
